media-sound/zynaddsubfx/zynaddsubfx-3.0.6-r1.ebuild

```diff
@@ -42,11 +49,22 @@ BDEPEND="
 PATCHES=(
 	"${FILESDIR}"/${P}-docs.patch
 	"${FILESDIR}"/${P}-stdint.patch
+	"${FILESDIR}"/${P}-libzest_location.patch
+)
+MRUBY_ZEST_PATCHES=(
+	"${FILESDIR}"/${PN}-mruby-zest-build-${PV}-cflags_ldflags.patch
+	"${FILESDIR}"/${PN}-mruby-zest-build-${PV}-libzest_location.patch
+	"${FILESDIR}"/${PN}-mruby-zest-build-${PV}-makefile_find.patch
+	"${FILESDIR}"/${PN}-mruby-zest-build-${PV}-system_wide_location.patch
```

Comment:
can't we just update those so that they can be applied using the standard PATCHES mechanism?

Reply:
Not particularly easily, no, because with the src provided, the tarball's paths aren't very nice.